### PR TITLE
AA Jurisdiction Progress screen (before audit launch)

### PR DESCRIPTION
**Pull request by jonahkagan**
**Description**

Adds a new screen where audit admins can view progress by jurisdiction (before the audit launches). Shows whether each jurisdiction has uploaded their ballot manifest or not.

This PR also renames `useParticipantsApi` to `useJurisdictions` and expands it with the full datatype.

**Testing**

Didn't add any new tests, existing tests pass.

**Progress**

Ready

### CODE REVIEW

File: arlo-client/src/components/Audit/Setup/Review/index.tsx [hunk 1]
```diff
@@ -40,22 +40,26 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
...
   const [{ contests }] = useContestsApi(electionId!, true)
   const targetedContests = contests
     .filter(c => c.isTargeted === true)
     .map(c => ({
       ...c,
       jurisdictionIds: c.jurisdictionIds.map(j =>
-        participants.length > 0 ? participants.find(p => p.id === j)!.name : ''
+        jurisdictions.length > 0
+          ? jurisdictions.find(p => p.id === j)!.name
+          : ''
```
Comment (benadida) [inline arlo-client/src/components/Audit/Setup/Review/index.tsx:52]: I think this is n^2 in the number of jurisdictions? Should we instead have an object of jurisdictions keyed by `id`? Also, the conditional on `jurisdictions.length` doesn't need to be inside the `map`.

File: arlo-client/src/components/Audit/Setup/Review/index.tsx [hunk 2]
```diff
@@ -40,22 +40,26 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
...
     }))
   const opportunisticContests = contests
     .filter(c => c.isTargeted === false)
     .map(c => ({
       ...c,
       jurisdictionIds: c.jurisdictionIds.map(j =>
-        participants.length > 0 ? participants.find(p => p.id === j)!.name : ''
+        jurisdictions.length > 0
+          ? jurisdictions.find(p => p.id === j)!.name
+          : ''
```
Comment (benadida) [inline arlo-client/src/components/Audit/Setup/Review/index.tsx:62]: same comment as previous.

File: arlo-client/src/components/Audit/Setup/Review/index.tsx (hunk 1 above)
Comment (jonahkagan) [reply arlo-client/src/components/Audit/Setup/Review/index.tsx:52]: I just renamed a variable here. do you want me to make this change?

Comment (benadida) [reply arlo-client/src/components/Audit/Setup/Review/index.tsx:52]: oh hah! I thought you'd introduced this. No, then maybe file a followup ticket to investigate this?

Comment (jonahkagan) [reply arlo-client/src/components/Audit/Setup/Review/index.tsx:52]: sure: https://github.com/votingworks/arlo/issues/474

**Pull request merged**